### PR TITLE
Fix/39038 import more fields

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/39038#note-54

The PR solves the issues found in the ticket note and adds a `tribe_remove_date_filters` query var support to the `Tribe__Events__Query::should_remove_date_filters` method.
The checks on the dates added to the query would create some false negatives on the check for existing posts.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Whitespace found at end of line | [File_Importer_Events.php:58](https://github.com/moderntribe/the-events-calendar/blob/71d369c87dfe827db076e27e7b90c3f27cf3ceef/src/Tribe/Importer/File_Importer_Events.php#L58)

> via jenkins/codesniffer-phpcs

File: tests/functional/Tribe/Events/Importer/File_Importer_Events_BooleanFieldsTest.php [hunk 1]
```diff
@@ -252,4 +252,40 @@ public function it_should_accept_falsy_values_to_show_map( $falsy_boolean_value
...
+			[ 'prefix', 'PREFIX' ],
+			[ 'postfix', 'postfix' ],
+			[ 'postfix', 'Postfix' ],
+			[ 'postfix', 'POSTFIX' ],
+			[ 'prefix', 'before' ],
+			[ 'prefix', 'Before' ],
+			[ 'prefix', 'BEFORE' ],
+			[ 'postfix', 'after' ],
+			[ 'postfix', 'After' ],
+			[ 'postfix', 'AFTER' ],
```
Comment (borkweb) [inline tests/functional/Tribe/Events/Importer/File_Importer_Events_BooleanFieldsTest.php]: In the UI, currency position uses the word `suffix`. I think that should be represented here and maybe replace `postfix`?

Comment (lucatume) [inline tests/functional/Tribe/Events/Importer/File_Importer_Events_BooleanFieldsTest.php]: Fixed, my mistake.

**Comment by borkweb** [comment]
:+1:

**Pull request merged**